source/solutions-library/rag-applications.txt

```diff
@@ -353,8 +305,8 @@ MongoDB Developer Data Platform
 Partner Technologies
 ~~~~~~~~~~~~~~~~~~~~
 
-- Confluent Cloud
-- AWS EC2
+- `Confluent Cloud <https://www.confluent.io/confluent-cloud/>`__
+- `Amazon EC2 <https://aws.amazon.com/pm/ec2/>`__
```

Comment:
Since these technologies are already mentioned at the top of the page, I think we can delete the entire "Learn more" section